### ISSUE TITLE
pandas 1.0 support

### ISSUE THREAD

**Issue by Triangulum9r** [holoviz/holoviz#248]
pandas gets downgraded when trying to install holoviz. When is 1.0 support coming?

```
conda install -c pyviz holoviz                                                 (compare-timeseries) 
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /Users/jremigio/anaconda3/envs/compare-timeseries

  added / updated specs:
    - holoviz


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    colorcet-2.0.2             |             py_0         1.5 MB  pyviz
    datashader-0.10.0          |             py_0        14.0 MB  pyviz
    holoviews-1.13.2           |             py_0         3.4 MB  pyviz
    param-1.9.3                |             py_0          77 KB  pyviz
    pyviz_comms-0.7.4          |             py_0          10 KB  pyviz
    ------------------------------------------------------------
                                           Total:        18.9 MB

The following NEW packages will be INSTALLED:

  appnope            pkgs/main/osx-64::appnope-0.1.0-py37_0
  attrs              pkgs/main/noarch::attrs-19.3.0-py_0
  backcall           pkgs/main/osx-64::backcall-0.1.0-py37_0
  bleach             pkgs/main/noarch::bleach-3.1.4-py_0
  bokeh              pkgs/main/osx-64::bokeh-2.0.1-py37_0
  bzip2              pkgs/main/osx-64::bzip2-1.0.8-h1de35cc_0
  cftime             pkgs/main/osx-64::cftime-1.1.1.1-py37h776bbcc_0
  click              pkgs/main/noarch::click-7.1.1-py_0
  cloudpickle        pkgs/main/noarch::cloudpickle-1.3.0-py_0
  colorcet           pyviz/noarch::colorcet-2.0.2-py_0
  curl               pkgs/main/osx-64::curl-7.69.1-ha441bb4_0
  cycler             pkgs/main/osx-64::cycler-0.10.0-py37_0
  cytoolz            pkgs/main/osx-64::cytoolz-0.10.1-py37h1de35cc_0
  dask               pkgs/main/noarch::dask-2.14.0-py_0
  dask-core          pkgs/main/noarch::dask-core-2.14.0-py_0
  datashader         pyviz/noarch::datashader-0.10.0-py_0
  datashape          pkgs/main/osx-64::datashape-0.5.4-py37_1
  decorator          pkgs/main/noarch::decorator-4.4.2-py_0
  defusedxml         pkgs/main/noarch::defusedxml-0.6.0-py_0
  distributed        pkgs/main/osx-64::distributed-2.14.0-py37_0
  entrypoints        pkgs/main/osx-64::entrypoints-0.3-py37_0
  fastparquet        pkgs/main/osx-64::fastparquet-0.3.2-py37h1d22016_0
  freetype           pkgs/main/osx-64::freetype-2.9.1-hb4e5f40_0
  fsspec             pkgs/main/noarch::fsspec-0.7.1-py_0
  hdf4               pkgs/main/osx-64::hdf4-4.2.13-h39711bb_2
  hdf5               pkgs/main/osx-64::hdf5-1.10.4-hfa1e0ec_0
  heapdict           pkgs/main/noarch::heapdict-1.0.1-py_0
  holoviews          pyviz/noarch::holoviews-1.13.2-py_0
  holoviz            pyviz/noarch::holoviz-0.11.3-py_0
  hvplot             pyviz/noarch::hvplot-0.5.2-py_0
  imageio            pkgs/main/noarch::imageio-2.8.0-py_0
  importlib_metadata pkgs/main/osx-64::importlib_metadata-1.5.0-py37_0
  ipykernel          pkgs/main/osx-64::ipykernel-5.1.4-py37h39e3cac_0
  ipython            pkgs/main/osx-64::ipython-7.13.0-py37h5ca1d4c_0
  ipython_genutils   pkgs/main/osx-64::ipython_genutils-0.2.0-py37_0
  ipywidgets         pkgs/main/noarch::ipywidgets-7.5.1-py_0
  jedi               pkgs/main/osx-64::jedi-0.16.0-py37_1
  jinja2             pkgs/main/noarch::jinja2-2.11.1-py_0
  jpeg               pkgs/main/osx-64::jpeg-9b-he5867d9_2
  jsonschema         pkgs/main/osx-64::jsonschema-3.2.0-py37_0
  jupyter_client     pkgs/main/noarch::jupyter_client-6.1.2-py_0
  jupyter_core       pkgs/main/osx-64::jupyter_core-4.6.3-py37_0
  kiwisolver         pkgs/main/osx-64::kiwisolver-1.1.0-py37h0a44026_0
  krb5               pkgs/main/osx-64::krb5-1.17.1-hddcf347_0
  libcurl            pkgs/main/osx-64::libcurl-7.69.1-h051b688_0
  libnetcdf          pkgs/main/osx-64::libnetcdf-4.7.3-he3b6227_0
  libpng             pkgs/main/osx-64::libpng-1.6.37-ha441bb4_0
  libsodium          pkgs/main/osx-64::libsodium-1.0.16-h3efe00b_0
  libssh2            pkgs/main/osx-64::libssh2-1.9.0-ha12b0ac_1
  libtiff            pkgs/main/osx-64::libtiff-4.1.0-hcb84e12_0
  llvmlite           pkgs/main/osx-64::llvmlite-0.31.0-py37h1341992_0
  locket             pkgs/main/osx-64::locket-0.2.0-py37_1
  markdown           pkgs/main/osx-64::markdown-3.1.1-py37_0
  markupsafe         pkgs/main/osx-64::markupsafe-1.1.1-py37h1de35cc_0
  matplotlib         pkgs/main/osx-64::matplotlib-3.0.3-py37h54f8f79_0
  mistune            pkgs/main/osx-64::mistune-0.8.4-py37h1de35cc_0
  msgpack-python     pkgs/main/osx-64::msgpack-python-1.0.0-py37h04f5b5a_1
  multipledispatch   pkgs/main/osx-64::multipledispatch-0.6.0-py37_0
  nbconvert          pkgs/main/osx-64::nbconvert-5.6.1-py37_0
  nbformat           pkgs/main/noarch::nbformat-5.0.4-py_0
  netcdf4            pkgs/main/osx-64::netcdf4-1.5.3-py37h28545a2_0
  networkx           pkgs/main/noarch::networkx-2.4-py_0
  notebook           pkgs/main/osx-64::notebook-6.0.3-py37_0
  numba              pkgs/main/osx-64::numba-0.48.0-py37h6c726b0_0
  olefile            pkgs/main/osx-64::olefile-0.46-py37_0
  packaging          pkgs/main/noarch::packaging-20.3-py_0
  pandoc             pkgs/main/osx-64::pandoc-2.2.3.2-0
  pandocfilters      pkgs/main/osx-64::pandocfilters-1.4.2-py37_1
  panel              pyviz/noarch::panel-0.9.5-py_0
  param              pyviz/noarch::param-1.9.3-py_0
  parso              pkgs/main/noarch::parso-0.6.2-py_0
  partd              pkgs/main/noarch::partd-1.1.0-py_0
  pexpect            pkgs/main/osx-64::pexpect-4.8.0-py37_0
  phantomjs          pyviz/osx-64::phantomjs-2.1.1-1
  pickleshare        pkgs/main/osx-64::pickleshare-0.7.5-py37_0
  pillow             pkgs/main/osx-64::pillow-7.0.0-py37h4655f20_0
  prometheus_client  pkgs/main/noarch::prometheus_client-0.7.1-py_0
  prompt-toolkit     pkgs/main/noarch::prompt-toolkit-3.0.4-py_0
  prompt_toolkit     pkgs/main/noarch::prompt_toolkit-3.0.4-0
  psutil             pkgs/main/osx-64::psutil-5.7.0-py37h1de35cc_0
  ptyprocess         pkgs/main/osx-64::ptyprocess-0.6.0-py37_0
  pyct               pyviz/noarch::pyct-0.4.6-py_0
  pyct-core          pyviz/noarch::pyct-core-0.4.6-py_0
  pygments           pkgs/main/noarch::pygments-2.6.1-py_0
  pyparsing          pkgs/main/noarch::pyparsing-2.4.6-py_0
  pyrsistent         pkgs/main/osx-64::pyrsistent-0.16.0-py37h1de35cc_0
  python-snappy      pkgs/main/osx-64::python-snappy-0.5.4-py37h0a44026_0
  pyviz_comms        pyviz/noarch::pyviz_comms-0.7.4-py_0
  pywavelets         pkgs/main/osx-64::pywavelets-1.1.1-py37h1de35cc_0
  pyyaml             pkgs/main/osx-64::pyyaml-5.3.1-py37h1de35cc_0
  pyzmq              pkgs/main/osx-64::pyzmq-18.1.1-py37h0a44026_0
  rise               pkgs/main/osx-64::rise-5.6.1-py37_0
  scikit-image       pkgs/main/osx-64::scikit-image-0.16.2-py37h6c726b0_0
  scipy              pkgs/main/osx-64::scipy-1.4.1-py37h9fa6033_0
  selenium           pkgs/main/osx-64::selenium-3.141.0-py37h1de35cc_0
  send2trash         pkgs/main/osx-64::send2trash-1.5.0-py37_0
  snappy             pkgs/main/osx-64::snappy-1.1.7-he62c110_3
  sortedcontainers   pkgs/main/osx-64::sortedcontainers-2.1.0-py37_0
  streamz            pkgs/main/noarch::streamz-0.5.2-py_0
  tbb                pkgs/main/osx-64::tbb-2020.0-h04f5b5a_0
  tblib              pkgs/main/noarch::tblib-1.6.0-py_0
  terminado          pkgs/main/osx-64::terminado-0.8.3-py37_0
  testpath           pkgs/main/noarch::testpath-0.4.4-py_0
  thrift             pkgs/main/osx-64::thrift-0.13.0-py37h0a44026_0
  toolz              pkgs/main/noarch::toolz-0.10.0-py_0
  tornado            pkgs/main/osx-64::tornado-6.0.4-py37h1de35cc_1
  traitlets          pkgs/main/osx-64::traitlets-4.3.3-py37_0
  typing_extensions  pkgs/main/osx-64::typing_extensions-3.7.4.1-py37_0
  wcwidth            pkgs/main/noarch::wcwidth-0.1.9-py_0
  webencodings       pkgs/main/osx-64::webencodings-0.5.1-py37_1
  widgetsnbextension pkgs/main/osx-64::widgetsnbextension-3.5.1-py37_0
  xarray             pkgs/main/osx-64::xarray-0.11.3-py37_0
  zeromq             pkgs/main/osx-64::zeromq-4.3.1-h0a44026_3
  zict               pkgs/main/noarch::zict-2.0.0-py_0
  zipp               pkgs/main/noarch::zipp-2.2.0-py_0
  zstd               pkgs/main/osx-64::zstd-1.3.7-h5bba6e5_0

The following packages will be DOWNGRADED:

  pandas                               1.0.3-py37h6c726b0_0 --> 0.24.2-py37h0a44026_0

```

**Comment by jbednar** [comment]
The `holoviz` metapackage typically gets updated when we do a major tutorial or training session, and the next two are currently scheduled for the end of April and for July.  We'll definitely do it before July, but I can't promise that we'll get it together in time for the one this month.  

In the meantime, you can install all the packages individually, using something like `conda install -c pyviz colorcet datashader holoviews hvplot panel snappy python-snappy streamz xarray`, which _should_ give you all the latest versions.  But I'm not sure which versions you'll get if you do that, and moreover we won't have tested that all the tutorial material is compatible with those versions, which is why we leave `holoviz` pinned at the last fully-tested version (and hence pandas<1.0).

**Comment by philippjfr** [comment]
>which is why we leave holoviz pinned at the last fully-tested version (and hence pandas<1.0).

This is not true, HoloViz is not pinned at all and currently broken in a myriad of ways.

**Comment by jbednar** [comment]
It was _meant_ to be pinned; alas. I released a new version 0.11.4 that was pinned properly to the older versions, and we now have a version 0.11.5 pinned to newer versions.
